src/vte.c

```diff
@@ -268,10 +268,11 @@ static void create_vte(void)
 	vc->menu = vte_create_popup_menu();
 	g_object_ref_sink(vc->menu);
 
-	frame = gtk_frame_new(NULL);
+	viewport = gtk_viewport_new (NULL, NULL);
+	gtk_viewport_set_shadow_type(GTK_VIEWPORT(viewport), GTK_SHADOW_NONE);
```

Comment:
same here, why not just a frame?  you can also set the frame's shadow type just fine with [`gtk_frame_set_shadow_type()`](https://developer.gnome.org/gtk2/stable/GtkFrame.html#gtk-frame-set-shadow-type)